pipeline_dp/dp_engine.py

```diff
@@ -420,10 +420,6 @@ def _check_aggregate_params(self,
         if check_data_extractors:
             _check_data_extractors(data_extractors)
         if params.contribution_bounds_already_enforced:
-            if data_extractors.privacy_id_extractor:
```

Comment:
Would it be possible to log a warning? Clients might have an incorrect configuration in this case.

Reply:
I'd say that having privacy_id_extractor is a correct configuration, even if no contribution bounding happens, e.g. in principle it would allow to compute PRIVACY_ID_COUNT per partition. So let's keep w/o warnings.